owncloudApp/src/main/java/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -42,6 +42,10 @@
 import static com.owncloud.android.domain.files.model.MimeTypeConstantsKt.MIME_PREFIX_TEXT;
 import static com.owncloud.android.domain.files.model.MimeTypeConstantsKt.MIME_PREFIX_VIDEO;
 
+@Deprecated
+/**
```

Comment:
I would use 

`@Deprecated("Use com.owncloud.android.domain.files.model.OCFile instead")`

Reply:
I wish i could, this is not kotlin :(

Reply:
![tenor](https://user-images.githubusercontent.com/15261363/97603089-5dc42f00-1a0c-11eb-8b65-158818293c81.gif)